src/cmd/sanity_test/sanity_tests.go

```diff
@@ -112,6 +119,76 @@ func checkEmailRestoration(client *msgraphsdk.GraphServiceClient, testUser, fold
 				*restore.GetTotalItemCount())
 			os.Exit(1)
 		}
+
+		checkAllSubFolder(user, restore, "", messageCount)
+	}
+}
+
+func getAllSubFolder(
```

Comment:
comments explaining what these functions do, if they test for anything, and what the conditions of the tests are, would be really helpful.